### ISSUE TITLE
Create tagging for the whole sentence in Custom NER model

### ISSUE THREAD

**Issue by Urvi04** [flairNLP/flair#2153]
I would like to train Flair's NER model to identify key sentences from the passage. I am not sure how the ColumnCorpus text should look like when I want the tag for the whole sentence and not just for the words within the sentence. 
For example, currently we have for the sentence "I like London and Berlin"
So, the train.txt will have:
I O
like O
London B-LOC
and O
Berlin B-LOC

But what if I want to tag this whole paragraph after splitting it to sentences as shown here:
https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_2_TAGGING.md#tagging-a-list-of-sentences 
Basically, something like this:
I like London and Berlin I-VIP
Who are you? O
